packages/ui/src/DateTimeActionSheet.tsx

```diff
@@ -105,7 +105,7 @@ function CalendarHeader({
   addMonth: (num: number) => void;
   month: Date[];
 }): React.ReactElement {
-  const displayDate = moment(month[0]).format("MMM YYYY");
+  const displayDate = moment.utc(month[0]).format("MMM YYYY");
```

Comment:
Confirming these changes are supposed to be in this PR?

Reply:
Nope, thank you! Testing out some weirdness with the dates.